keel-clouddriver/src/main/kotlin/com/netflix/spinnaker/keel/clouddriver/MemoryCloudDriverCache.kt

```diff
@@ -44,38 +49,63 @@ import retrofit2.HttpException
  * For more details on why using async, see: https://github.com/spinnaker/keel/pull/1154
  */
 class MemoryCloudDriverCache(
-  private val cloudDriver: CloudDriverService
+  private val cloudDriver: CloudDriverService,
+  private val meterRegistry: MeterRegistry
 ) : CloudDriverCache {
 
   private val securityGroupSummariesByIdOrName = Caffeine.newBuilder()
+    .executor(IO.asExecutor())
```

Comment:
I made all the caches use this executor as they do network IO to talk to Clouddriver